pytorch/training/docker/1.12/py3/cu113/Dockerfile.gpu

```diff
@@ -456,9 +459,10 @@ RUN mkdir -p /etc/pki/tls/certs && cp /etc/ssl/certs/ca-certificates.crt /etc/pk
 WORKDIR /
 
 # Install scikit-learn and pandas
-RUN conda install -y -c conda-forge \
+RUN /opt/conda/bin/conda install -y -c conda-forge \
     scikit-learn \
-    pandas
+    pandas \
+ && /opt/conda/bin/conda clean -ya
```

Comment:
What does `conda clean` do? Why do we need it?

Reply:
initially thought to remove the root cache, but just removed it!